cmake/templates/cvconfig.h.in

```diff
@@ -248,4 +248,7 @@
 /* Library QR-code decoding */
 #cmakedefine HAVE_QUIRC
 
+/* Tengine support */
+#cmakedefine HAVE_TENGINE
+
```

Comment:
Please avoid updating of common cvconfig.h.
This is DNN specific stuff so please use `add_definition()` in `modules/dnn/CMakeLists.txt` instead.

Reply:
done